go.mod

```diff
@@ -203,7 +203,10 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
-require github.com/google/go-containerregistry v0.8.0
+require (
+	github.com/google/go-containerregistry v0.8.0
+	github.com/samber/lo v1.27.0
```

Comment:
@jLopezbarb and @odedlaz  I have a strong preference for not bringing in this dependency. It is quite large and we are only using what looks like a small portion of it. Adding a dependency can add a lot of long-term overhead and in this case I believe we can have similar functionality without the dependency.

Reply:
I'll remove it to ease the PR approval. It does remove a lot of boilerplate code though.

I'd consider adding it in the future.

Reply:
@jmacelroy , remove the [samber/lo](https://github.com/samber/lo) dependency.